apis/printservice/printservice.yaml

```diff
@@ -0,0 +1,211 @@
+        post:
+            description:  >-
+                This API allows you to get pdf from html link.
+                   - The **__/v1/print/preview/generate__** endpoint executes the "generate pdf" request based on **query** params provided in the request body.
+                   - All fields marked with an * are mandatory
+                   - Mandatory fields cannot be null or empty
+            summary: generating pdf
+            tags:
+                - PDF Preview API
+            operationId: preview PDF
+            produces:
+                - application/json
+            parameters:
+                - in: query
+                  name: fileUrl
+                  type: string
+                  description: For example https://www.example.com, url of html which needs to be converted to pdf.
+                  required: true
+            responses:
+                '200':
+                    description: 'OK ! Operation successful."Certificate addition" operation was executed successfully.'
+                    schema:
+                        allOf:
+                            - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+                            - $ref: '#/definitions/PrintPdfResponse'
+                '400':
+                    description: 'BAD REQUEST. The "Certificate addition" operation failed. Input for a mandatory parameter may be missing.'
+                    schema:
+                        $ref: '#/definitions/PdfGenErrorResponse'
+                '500':
+                    description: 'INTERNAL SERVER ERROR ! Looks like something went wrong. These errors are tracked automatically. Try refreshing the page. If the problem persists, contact info@sunbird.org. '
+
+
+definitions:
+
+    PdfGenerationRequest:
+        title: Certificate Verify request
+        type: object
+        properties:
+            request:
+                $ref: '#/definitions/genPdfRequest'
+        required:
+            - request
+
+    genPdfRequest:
+        type: object
+        description: htmlTemplate zip url should be provided always while making requests.
+        properties:
+            htmlTemplate:
+                description: The htmlTemplate zip url should contain index.html(with placeholders) and its related css & images file, which are required to generate certificate.
+                type: string
+            context:
+                description: This will contain the placeHolder and it's value which needs to be replaced in HTML.
+                type: object
+                example:
+                    recipientName: John Deo
+                    certificateName: ABC Certificate
+            storageParams:
+                type: object
+                description: this object will have the container name and path where its needs to be stored in cloud. it not provided it will stored at default location.
```

Comment:
@anmol2302 modify the sentence to, 'This object has the container name and path where the certificate needs to be stored. If this is not provided, the system stores the certificate in the default location.